apps/emqx_bridge_tablestore/rebar.config

```diff
@@ -0,0 +1,7 @@
+{erl_opts, [debug_info]}.
+{deps, [
+    {ots_erl, {git, "https://github.com/emqx/ots_erl.git", {tag, "0.2.2"}}},
```

Comment:
this dependency needs a better readme and a github repo description.

Reply:
Improved the README.md